src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.motors.PIDSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+@RobotMapFor("Gaston")
+public class GastonMap extends RobotMap {
+    // These are outsied the methods to prevent resource leaks
+    private final DigitalInput leftUpperLimit = new DigitalInput(0);
+    private final DigitalInput leftLowerLimit = new DigitalInput(1);
+    private final DigitalInput rightUpperLimit = new DigitalInput(2);
+    private final DigitalInput rightLowerLimit = new DigitalInput(3);
+
+    public TelescopeMap getLeftTelescopeMap() {
+        TelescopeMap map = new TelescopeMap(new PIDSparkMax(0, MotorType.kBrushless), leftUpperLimit::get,
```

Comment:
Let's create the `PIDSparkMax` object outside of this like you did for the Digital Inputs. We're likely to need to adjust settings on them.